src/clr-addons/brand-avatar/brand-avatar.scss

```diff
@@ -0,0 +1,28 @@
+@include exports('brand-avatar.clr-addons') {
+  .avatar-container {
```

Comment:
pls nest everything inside .brand-avatar and set this class to the component selector via host binding inside @Component.
this ensures that the css doesn't get polluted.
see e.g. pager.ts and the class "pager"

Reply:
done